index.js

```diff
@@ -27,15 +27,36 @@ module.exports.onRendererWindow = async (window) => {
   }
 
   const config = window.config.getConfig();  
-  let deg = 0;
+  
+  if (config.hyperBorder && config.hyperBorder.animate) {
 
-  const updateBorderAngle = () => {
-    window.document.documentElement.style.setProperty('--border-angle', `${deg}deg`);
-    deg = deg > 360 ? 2 : deg + 2;
-  };
+    let deg = 0;
+    let intervalId = null;
+    
+    const updateBorderAngle = () => {
+      window.document.documentElement.style.setProperty('--border-angle', `${deg}deg`);
+      deg = deg > 360 ? 2 : deg + 2;
+    };
 
-  if (config.hyperBorder && config.hyperBorder.animate) {
-    window.setInterval(updateBorderAngle, 100);
+    const createUpdateBorderAngleInterval = () => {
+      if (!intervalId) {
+        intervalId = window.setInterval(updateBorderAngle, 100);        
+      }
+    };
+
+    const clearUpdateBorderAngleInterval = () => {
+      if (intervalId) {
+        window.clearInterval(intervalId);
+        intervalId = null;
+      }
+    };
+
+    browserWindow.on('close', clearUpdateBorderAngleInterval );
```

Comment:
These 3 lines have an inconsistent padded space before the closing paren

Reply:
fixed